src/main/java/domain/WinnerRanking.java

```diff
@@ -0,0 +1,24 @@
+package domain;
+
+public enum WinnerRanking {
+    RANK_ONE(2000000000,6 ),
+    RANK_TWO(1500000, 5),
+    RANK_THREE(50000, 4),
+    RANK_FOUR(5000, 3);
```

Comment:
Java 7부터는 숫자 가독성을 높이기 위해 아래처럼 사용을 할 수 있습니다:)
```suggestion
    RANK_ONE(2_000_000_000,6 ),
    RANK_TWO(1_500_000, 5),
    RANK_THREE(50_000, 4),
    RANK_FOUR(5_000, 3);
```

Reply:
정보 감사합니다👍